tests/suites/dashboard/dashboard.sh

```diff
@@ -17,7 +17,7 @@ run_dashboard_deploy() {
 
 	destroy_model "${model_name}"
 	juju switch controller
-	juju remove-application dashboard
+	juju remove-application --no-prompt dashboard
```

Comment:
I don't think this is a correct fix. We pass on future branches, and on aws

Reply:
Test passes (on my machine) without. I think the greater timeout is all that's needed

Reply:
Yea, seems correct, we must be saying yes automatically to these prompts somewhere.

Reply:
Will leave this in though because it was annoying me when testing locally, if thats alright with you

Reply:
I'm very surprised this makes any difference for you

Reply:
I would rather it not be, since it's misleading, but I see it has been added in other places, so I'll allow it here

In future, this could do with a deeper look